### PR TITLE
Create ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by ensu

Ticket: [MTM-57348](https://cumulocity.atlassian.net/browse/MTM-57348)
Original PR: [5313](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5313)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Ng-cli name, context-path and key are updated automatically when they are not present.
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Application name, context path and key automatically updated when missing
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57348
+version: 1020.0.0
+---
+Ng-cli name, context-path and key are updated automatically when they are not present.
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, when creating a new application in the UI, the application name, context path and key fields were not automatically populated, requiring manual input from the user. With this change, if the application name, context path or key are not provided during application creation, they will be automatically derived from the uploaded application file name. This improves the user experience by reducing the amount of manual input required and ensures that applications can be created successfully even if some fields are left empty.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md (hunk 1 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md:3]:
```suggestion
title: Ng-cli name, context-path and key are updated automatically when they are not present in the "cumulocity.config.ts" file.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md (hunk 2 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md:17]:
```suggestion
Ng-cli name, context-path and key are updated automatically when they are not present in the "cumulocity.config.ts" file. If they are not present they will be derived from the "name" property in the "package.json" file
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57348
+version: 1020.0.0
+---
+Ng-cli name, context-path and key are updated automatically when they are not present in the "cumulocity.config.ts" file. If they are not present they will be derived from the "name" property in the "package.json" file.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md:17]:
```suggestion
If `ng-cli name`, `context-path` and `key` are not present in the *cumulocity.config.ts* file they are automatically derived from the `name` property in the *package.json* file.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57348
+version: 1020.0.0
+---
+If `ng-cli name`, `context-path` and `key` are not present in the *cumulocity.config.ts* file they are automatically derived from the `name` property in the *package.json* file.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md:17]:
```suggestion
If values for `ng-cli name`, `context-path` and `key` are missing in the *cumulocity.config.ts* file they are automatically derived from the `name` property in the *package.json* file.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Ng-cli name, context-path and key are updated automatically when they are not present in the "cumulocity.config.ts" file.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md:3]:
```suggestion
title: Missing values in cumulocity.config.ts file automatically added
```

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Missing values in cumulocity.config.ts file automatically added
```
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-ng-cli-name-context-path-and-key-are-updated-automatically-when.md:3]:
```suggestion
title: Missing properties in cumulocity.config.ts file automatically added
```

**Pull request merged**